config/parsers/mojang.ts

```diff
@@ -0,0 +1,9 @@
+export default function (response: {
```

Comment:
This still needs to be changed. Take a look at my previous comments ([here](https://github.com/IgnacioBarocchi/HumbleScraper/pull/5#discussion_r445206551) and [here](https://github.com/IgnacioBarocchi/HumbleScraper/pull/5#discussion_r445808621)):

```typescript
import { Response } from 'node-fetch';

export = function (response: Response): void {
  console.log(`response url: ${decodeURIComponent(response.url)}`);
  console.log(`response status: ${response.status}`);
  console.log(`response status text: ${response.statusText}`);
};
```

The parsers need to expect a response argument of type `Response`.
The type you used, which is this:
```
  url: string;
  status: number;
  statusText: string;
```
is compatible, because it has some of the object keys, but it's not a 100% match with what Request has. If, in the future we want to use, for example, `request.body`, we'll need to come back here and update the `type`. And if we have multiple parsers, we'll need to do it for all of them! 

If we use `Response`, which is the actual type we're sending when calling this function in `parser(response)` in `index.ts`, we won't have to do this.

Reply:
👍 👌